RELEASE-NOTES.rst

```diff
@@ -18,6 +18,55 @@
 Release notes
 =============
 
+v0.5.1
+------
+
+-  User interface:
+
+   -  Adds a new method to change session locale, in preparation for the
+      future pure JS application.
+   -  Rewrites frontpage to emphasize the public demo site and improves
+      it rendering on small screens.
+   -  Improves the search input suggestion UX.
+
+-  Fixtures: updates the third organisation default circulation policy
+   for the workshops.
+-  Documentation:
+
+   -  Improves the github issue template with information on the
+      instance and version of RERO ILS on which the issue occurred.
+
+-  Fixed issues:
+
+   -  `#488 <https://github.com/rero/rero-ils/issues/488>`__: disallows
+      editing libraries by a non affiliated librarian.
+   -  `#475 <https://github.com/rero/rero-ils/issues/475>`__: redirects
+      homepage to the global view.
+   -  `#540 <https://github.com/rero/rero-ils/issues/540>`__: helps
+      distinguish organisation from organisation as an author in order
```

Comment:
distinguish "organisation" as a RERO ILS client from "organisation" as an author, in order